### DISCUSSION TITLE
Data persistence

### DISCUSSION THREAD

**Discussion by Naegionn** [nextcloud/all-in-one#4780] [Questions] [answered]
Hey 
How is data persitence handled for nextcloud aio?

The "user" data such as images are stored on a volume provided to the nextcloud container.
How is the nextcloud configuration, e.g. user accounts, shares themes.. persistet?

There is a database container but i dont see it having a persitent volume.

Thanks for your help

**Comment by szaimen** [comment, marked as answer]
Hi, if you are able to read docker-compose syntax, here is hos things are set up internally: https://github.com/nextcloud/all-in-one/blob/main/manual-install/latest.yml
